tests/sentry/runner/test_initializer.py

```diff
@@ -243,7 +246,11 @@ def test_apply_legacy_settings(settings):
             ("SENTRY_ENABLE_EMAIL_REPLIES", "SENTRY_OPTIONS['mail.enable-replies']"),
             ("SENTRY_FILESTORE", "SENTRY_OPTIONS['filestore.backend']"),
             ("SENTRY_FILESTORE_OPTIONS", "SENTRY_OPTIONS['filestore.options']"),
-            ("SENTRY_FILESTORE_RELOCATION", "SENTRY_OPTIONS['filestore.relocation']"),
+            ("SENTRY_RELOCATION_FILESTORE", "SENTRY_OPTIONS['filestore.relocation-backend']"),
+            (
+                "SENTRY_RELOCATION_FILESTORE_OPTIONS",
+                "SENTRY_OPTIONS['filestore.relocation-options']",
```

Comment:
so filestore.relocation-backend should be `sentry.filestore.gcs.GoogleCloudStorage`
and filestore.relocation-options should be `sentry-us-silo-relocation`
correct?